### PR TITLE
upgraded to next-9.1.1

### PR DESCRIPTION
**Pull request by amrit07**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -144,7 +144,7 @@
     "invariant": "^2.2.4",
     "isomorphic-unfetch": "^3.0.0",
     "lodash": "^4.17.11",
```
Comment (tirthaguha) [inline package.json]: take out isometric-unfetch, upgrade react, react-dom to v16.10

**Comment by vinodloha** [comment]
> the package json needs further cleanup

Great to have your support here @tirthaguha :)

File: package.json (hunk 1 above)
Comment (vinodloha) [reply package.json]: Will need isometric-unfetch for SSR uses of fetch. Updated React and React DOM. 

Futher package updates and cleanup in next PR.

**Comment by vinodloha** [comment]
> isometric-unfetch, and axios together in the same repo doesn't make much sense,

Agreed, will get rid of axios 🗡

**Pull request merged**